mythril/plugin/discovery.py

```diff
@@ -12,21 +12,30 @@ class PluginDiscovery(object, metaclass=Singleton):
     """
 
     # Installed plugins structure. Retrieves all modules that have an entry point for mythril.plugins
-    _installed_plugins = {
-        entry_point.name: entry_point.load()
-        for entry_point in pkg_resources.iter_entry_points("mythril.plugins")
-    }
+    _installed_plugins = {}  # type: Dict[str, Any]
```

Comment:
```suggestion
    _installed_plugins = None  # type: Optional[Dict[str, Any]]
```

Reply:
Without something like this we could be running the init_installed_plugins forever rite?